coding-guides/tracking-used-invites.md

```diff
@@ -28,10 +28,9 @@ const wait = require('util').promisify(setTimeout);
 client.on('ready', async () => {
   // "ready" isn't really ready. We need to wait a spell.
   await wait(1000);
```

Comment:
Since we are foreced to use node v16 we now have access to the async timers api. We could potentially update the 
```js
const wait = require('util').promisify(setTimeout);
```
to use
```js
const wait = require('timers/promises').setTimeout;
``` 
See https://nodejs.org/api/timers.html#timers_timerspromises_settimeout_delay_value_options.